api/spec/controllers/mno_enterprise/jpi/v1/admin/organizations_controller_spec.rb

```diff
@@ -202,11 +202,24 @@ module MnoEnterprise
       it { expect(response).to be_success }
     end
 
+    describe 'GET #download_batch_example' do
+
+      subject { get :download_batch_example, file: file }
+
+      context 'successful download' do
+        let(:path) { MnoEnterprise::Api::Engine.root.join('spec/fixtures/batch-download-example.csv') }
```

Comment:
Do you need to introduce an extra file here?
Why not reuse `MnoEnterprise::Api::Engine.root.join('app/assets/batch-example.csv')`?